tuf/store/filestore.go

```diff
@@ -49,7 +49,7 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	if size == -1 {
+	if size == MaxSize {
```

Comment:
Since we have already change `-1` to the constant `MaxSize `, could we also update the comments for this function.

The old comments is try to explain what's the `-1` stand for, and I think it might be out of date now.